recipes/zstash/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "zstash" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/zstash/archive/v{{ version }}.tar.gz
+  sha256: 16d4169dcf49057ac86fb78410bffbc9ceba7680f73d4f88659992c18376cb63
+
+build:
+  entry_points:
+    - zstash=zstash.main:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - six ==1.16.0
+    - globus-sdk <4.0.0,>=3.0.0
+    - fair-research-login ==0.2.6
```

Based on https://github.com/E3SM-Project/zstash/blob/main/conda/meta.yaml#L24-L26
we don't need to constrain six or fair-research-login.  The globus constrain is also different.
```suggestion
    - six
    - globus-sdk >=2
    - fair-research-login
```

---

@forsyth2 and @tomvothecoder, is this not where you keep the requirements you use to build the conda package now?

---

They're in the `conda` subdirectory: https://github.com/E3SM-Project/zppy/tree/main/conda, https://github.com/E3SM-Project/zstash/tree/main/conda

---

The `conda` subdirectory is correct: https://github.com/E3SM-Project/zppy/tree/main/conda, https://github.com/E3SM-Project/zstash/tree/main/conda.

---

Okay, @forsyth2, I will make a PR to fix `setup.py`, then.